### ISSUE TITLE
Multi-platform prebuilds?

### ISSUE THREAD

**Issue by H4M5TER** [prebuild/prebuild#279]
Is it expectable that I only generated tarballs on platform of running system?
For `prebuild -t 12.0.0 -t 14.0.0 -t 16.0.0 --strip` it only generated `node-v72/v83/v93-win32-x64.tar.gz`.
Are they able to be used on other platforms?
It's very weird since [prebuildify](https://github.com/prebuild/prebuildify) has provide platform option.

**Comment by vweevers** [comment]
> Are they able to be used on other platforms?

They're only suitable for the platform you're building on. Most folks use CI to build for other platforms.

> It's very weird since prebuildify has provide platform option.

It's the same for prebuildify; the platform option by itself isn't enough to cross-compile.

**Comment by H4M5TER** [comment]
Thank you for answering that.
I tried to build both windows and linux using prebuildify on the same machine on CI before I see your reply.
And it not work, files on `win32-x64` are exactly same with files on `linux-x64`.
I searched the web and it seems that [node-gyp doesn't support cross-compiling](https://github.com/nodejs/node-gyp/issues/829).
If it's no possible to cross-compile, what's the use of platform option of prebuildify since there is a default value of  `os.platform()`?
If it's possible, what should I do to cross-compile? Such as replace compile toolchain on the PATH with mingw, modifying `binding.gyp`, or whatever.
